interface/templates/interface/users_list.html

```diff
@@ -23,12 +31,19 @@ <h5><b>Scor maxim:</b> {{ assignment.max_score }}</h5>
           <tr>
             <td><a href="{% url 'subs_for_user' sub.assignment.course.pk sub.assignment.pk sub.user.username %}">{{ sub.user.username }}</a></td>
             <td>{{ sub.timestamp|date:'Y-m-d H:i' }}</td>
-            <td>{{ sub.total_score }}</td>
+            <td>{{ sub.score }}</td>
+            {% if not sub.assignment.hidden_score and sub.user.username == user.username or not sub.assignment.hidden_score and user in sub.assignment.course.teaching_assistants.all %}
```

Comment:
Django templates should allow the use of parenthesis so this can become `not sub.assignment.hidden_score and (sub.user.username == user.username or user in sub.assignment.course.teaching_assistants.all)`